gql/resolver_versions.go

```diff
@@ -321,7 +319,27 @@ func (r *getVersionsResolver) Count(ctx context.Context, _ *generated.GetVersion
 
 type versionResolver struct{ *Resolver }
 
-func (r *versionResolver) Link(_ context.Context, obj *generated.Version) (string, error) {
+func findWindowsTarget(obj *generated.Version) *generated.VersionTarget {
+	var windowsTarget *generated.VersionTarget
+	for _, target := range obj.Targets {
+		if target.TargetName == "Windows" {
+			windowsTarget = target
+			break
+		}
+	}
+	return windowsTarget
+}
```

Comment:
Could we instead keep storing the link hash and size of the windows version into the database for those fields?

This code assumes that the targets will always get selected in GQL which is not guaranteed.

Reply:
Since `Version.Key` contains the link to the original uploaded file, I was thinking the `Size` and `Hash` should be the ones for that file.
And while looking for the usages of `Version.Hash` and `.Size`, I found that they are also updated by `consumers.UpdateModDataFromStorage`, which should also update the targets.